frontend/src/screens/Home.tsx

```diff
@@ -59,7 +60,15 @@ function Home() {
 
   return (
     <>
-      <AppHeader title={getGreeting(albyMe?.name)} description="" />
+      <AppHeader
+        title={getGreeting(albyMe?.name)}
+        description=""
+        contentRight={
+          <UpgradeDialog>
+            <Button variant="premium">Upgrade</Button>
```

Comment:
![image](https://github.com/user-attachments/assets/9eee2ec6-8d6c-4078-9c32-617d14631bf1)

Does the upgrade button feel strange here? I wonder if later we could move it to a card on the home page to explain the benefits that users can see before they click?

Reply:
also that removes duplication on other pages (e.g. on the alby jim page the button is shown twice)